### ISSUE TITLE
AttributeError: module 'torch.nn.functional' has no attribute 'interpolate'

### ISSUE THREAD

**Issue by ysduxx34** [xxradon/PytorchToCaffe#19]
python 0.4.0

**Comment by xxradon** [comment]
Yes,[in this ](https://pytorch.org/docs/0.4.1/nn.html?highlight=interpolate#torch.nn.functional.interpolate),interpolate is adoptted by pytorch 0.4.1 and later version.
And I think you should update your pytorch version.
